src/features/list-view/examples/ListViewExamples.js

```diff
@@ -15,6 +15,13 @@ const getGridHeader = columnIndex => `Header ${generateRandomString(columnIndex)
 
 const getGridCell = ({ columnIndex, rowIndex }) => `Row ${rowIndex}, Column ${columnIndex}`;
 
+const isGridHeaderSortable = () => true;
+
+const onSortChange = columnIndex => {
+    // eslint-disable-next-line no-console
+    console.log(`Column header #${columnIndex} clicked`);
```

Comment:
would you mind explaining the reason we are comitting a console log to the code base?

Reply:
It's part of styleguidist -> Log to console when user clicks on header [to sort].